src/ui/components/search/searchcomponent.js

```diff
@@ -602,6 +600,7 @@ export default class SearchComponent extends Component {
     this._throttled = true;
     setTimeout(() => { this._throttled = false; }, this._searchCooldown);
 
+    this.customHooks.onConductSearch(query);
```

Comment:
Why here and not in the `search` function?

Reply:
Ah, good point